hazelcast/src/main/java/com/hazelcast/internal/partition/impl/MigrationStats.java

```diff
@@ -17,48 +17,59 @@
 package com.hazelcast.internal.partition.impl;
 
 import com.hazelcast.internal.metrics.Probe;
-import com.hazelcast.internal.util.Clock;
 import com.hazelcast.internal.partition.MigrationStateImpl;
+import com.hazelcast.internal.util.Clock;
 import com.hazelcast.partition.MigrationState;
 
 import java.util.Date;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicInteger;
 import java.util.concurrent.atomic.AtomicLong;
 
+import static com.hazelcast.internal.metrics.MetricDescriptorConstants.MIGRATION_METRIC_COMPLETED_MIGRATIONS;
+import static com.hazelcast.internal.metrics.MetricDescriptorConstants.MIGRATION_METRIC_ELAPSED_DESTINATION_COMMIT_TIME;
+import static com.hazelcast.internal.metrics.MetricDescriptorConstants.MIGRATION_METRIC_ELAPSED_MIGRATION_OPERATION_TIME;
+import static com.hazelcast.internal.metrics.MetricDescriptorConstants.MIGRATION_METRIC_ELAPSED_MIGRATION_TIME;
+import static com.hazelcast.internal.metrics.MetricDescriptorConstants.MIGRATION_METRIC_LAST_REPARTITION_TIME;
+import static com.hazelcast.internal.metrics.MetricDescriptorConstants.MIGRATION_METRIC_PLANNED_MIGRATIONS;
+import static com.hazelcast.internal.metrics.MetricDescriptorConstants.MIGRATION_METRIC_TOTAL_COMPLETED_MIGRATIONS;
+import static com.hazelcast.internal.metrics.MetricDescriptorConstants.MIGRATION_METRIC_TOTAL_ELAPSED_DESTINATION_COMMIT_TIME;
+import static com.hazelcast.internal.metrics.MetricDescriptorConstants.MIGRATION_METRIC_TOTAL_ELAPSED_MIGRATION_OPERATION_TIME;
+import static com.hazelcast.internal.metrics.MetricDescriptorConstants.MIGRATION_METRIC_TOTAL_ELAPSED_MIGRATION_TIME;
+
 /**
  * Collection of stats for partition migration tasks.
  */
 public class MigrationStats {
 
-    @Probe
+    @Probe(name = MIGRATION_METRIC_LAST_REPARTITION_TIME)
```

Comment:
How about `MS` unit here and in other time related probes in this class?

Reply:
Added and introduced `NS` to the `ProbeUnit` enum for nanoseconds.